src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/ftacommunication/FtaCommunicationAboutToSubmitHandler.java

```diff
@@ -0,0 +1,94 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+import uk.gov.hmcts.reform.sscs.idam.IdamService;
+import uk.gov.hmcts.reform.sscs.idam.UserDetails;
+
+
+@Service
+@Slf4j
+public class FtaCommunicationAboutToSubmitHandler implements PreSubmitCallbackHandler<SscsCaseData> {
+    private IdamService idamService;
+
+    @Autowired
+    public FtaCommunicationAboutToSubmitHandler(IdamService idamService) {
+        this.idamService = idamService;
+    }
+
+    @Override
+    public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
+        requireNonNull(callback, "callback must not be null");
+        requireNonNull(callbackType, "callbacktype must not be null");
+
+        return callbackType.equals(CallbackType.ABOUT_TO_SUBMIT)
+            && callback.getEvent() == EventType.FTA_COMMUNICATION;
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback,
+                                                          String userAuthorisation) {
+
+        if (!canHandle(callbackType, callback)) {
+            throw new IllegalStateException("Cannot handle callback");
+        }
+
+        CaseDetails<SscsCaseData> caseDetails = callback.getCaseDetails();
+        SscsCaseData sscsCaseData = caseDetails.getCaseData();
+
+        FtaCommunicationFields ftaCommunicationFields = Optional.ofNullable(sscsCaseData.getFtaCommunicationFields())
+            .orElse(FtaCommunicationFields.builder().build());
+
+        String topic = ftaCommunicationFields.getFtaRequestTopic();
+        String question = ftaCommunicationFields.getFtaRequestQuestion();
+        LocalDateTime now = LocalDateTime.now();
+        LocalDateTime dueDate = calculateDueDate(LocalDateTime.now());
+        final UserDetails userDetails = idamService.getUserDetails(userAuthorisation);
+
+        List<FtaCommunication> ftaComs = Optional.ofNullable(ftaCommunicationFields.getFtaCommunications())
+            .orElse(new ArrayList<>());
+
+        ftaComs.add(FtaCommunication.builder()
+            .value(FtaCommunicationDetails.builder()
+                .requestText(question)
+                .requestTopic(topic)
+                .requestDateTime(now)
+                .requestUserName(userDetails.getName())
+                .requestDueDate(dueDate)
+                .build())
+            .build());
+
+        ftaComs.sort(Comparator.comparing(ftaCom ->
+            ((FtaCommunication) ftaCom).getValue().getRequestDateTime()).reversed());
+
+        ftaCommunicationFields.setFtaCommunications(ftaComs);
```

Comment:
between line 77 and 78 (inside the above mentioned if statement) add in 
```
ftaCommunicationFields.setFtaCommunicationFilter("awaitingInfoFromFta");
ftaCommunicationFields.setTribunalCommunicationFilter("provideInfoToTribunal");
```
(use enum values if done in commons)